.github/workflows/compliance.yml

```diff
@@ -0,0 +1,34 @@
+name: Compliance
+
+on:
+  push:
+    branches: [ main ]
+  pull_request: {}
+
+permissions:
+  # https://docs.github.com/en/rest/overview/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-contents
+  contents: read
+
+jobs:
+  go:
```

Comment:
Conflicts with a job name from `go.yml`, which is a problem for branch protection rules on GitHub, see #113.